lib/schema/createModelsFromSchema.js

```diff
@@ -10,10 +10,11 @@ exports.createModelsFromSchema = function () {
 
 	Object.keys(self.schema.objects).forEach(function (modelName) {
 		var object = self.schema.objects[modelName],
+			primaryKeyName = self.schema.primary_keys[modelName] || 'ID',
```

Comment:
It's not strictly necessary to have a primary key. I think we should handle this case correctly instead of just assuming the primary key to be ID.